ledger/src/snapshot_package.rs

```diff
@@ -1,5 +1,6 @@
 use solana_runtime::accounts_db::AccountStorageEntry;
 use solana_runtime::status_cache::SlotDelta;
+use solana_sdk::clock::Slot;
```

Comment:
@ryoqun - for next time, please just break out mechanical changes like `s/u64/Slot` in `ledger/src/snapshot_package.rs` into its own individual PR.  This kind of thing is very quick and easy to review (if even needed!), and can land independently so it doesn't clog up a larger PR

Reply:
I see! Reverted fa737f6bece7a5ddd06fb02ee8287b376bc345df